tools/test/h5gentest.c

```diff
@@ -275,7 +275,7 @@ gen_h5dump_files(void)
     gent_trefer_reg_1d();
 
     nerrors += gent_test_reference_external();
-
+    nerrors += (gent_tvms() < 0 ? 1 : 0);
```

Comment:
What does vms stand for?,

Reply:
Most likely "Virtual Memory System" from VAX/VMS